js/views/home.vue

```diff
@@ -40,7 +41,8 @@ export default {
             metrics: new MyMetrics(),
             reuses: new PageList({
                 ns: 'me',
-                fetch: 'my_org_reuses'
+                fetch: 'my_org_reuses',
```

Comment:
Yes, maybe in another PR on `org` vs `organization` naming convention (this one require to change on the API side too)